netcat/README.md

```diff
@@ -0,0 +1,57 @@
+# netcat command
+
+This is a simple `netcat` command.
+It's meant to be used with the binary-compatibility mode of Unikraft.
+
+The contents are:
+
+* /usr/bin/netcat: pre-built Linux ELF
+* lib/, lib64/: pre-extracted dynamic libraries required to run the netcat
+
+
+## (Re)Extract Dynamic Libraries
+
+You can see the list of dynamics libraries for the `netcat` command using the `ldd` command:
+
+```console
+$ ldd /usr/bin/netcat
+
```

Comment:
Separate the command from its output. Remove the `$` prompt sign to make it easy to copy-paste.